opm/common/OpmLog/LogBackend.cpp

```diff
@@ -62,13 +62,19 @@ namespace Opm {
 
         // Use the message limiter (if any).
         MessageLimiter::Response res = m_limiter
-            ? m_limiter->handleMessageTag(messageTag)
+            ? m_limiter->handleMessageLimits(messageTag, messageFlag)
             : MessageLimiter::Response::PrintMessage;
-        if (res == MessageLimiter::Response::JustOverLimit) {
+        if (res == MessageLimiter::Response::JustOverTagLimit) {
             // Special case: add a message to this backend about limit being reached.
             std::string msg = "Message limit reached for message tag: " + messageTag;
             addTaggedMessage(messageFlag, "", msg);
         }
+        if (res == MessageLimiter::Response::JustOverCategoryLimit) {
+            // Special case: add a message to this backend about limit being reached.
+            std::string msg = "Message limit reached for message : " + Log::prefixMessage(messageFlag, "");
+            addTaggedMessage(messageFlag, "", msg);
```

Comment:
>Then addTaggedMessage can simply call includeMessage() and then (if true is returned) call unconditionallyAddMessage.

I don't quite understand this. For example, in ``StreamLog``, if true is returned, the messages will be output. So what's the meaning of that function, I mean``unconditionallyAddMessage``?